Packs/AlibabaActionTrail/Integrations/AlibabaActionTrailEventCollector/README.md

```diff
@@ -7,18 +7,18 @@ This integration was integrated and tested with API version 0.6 of Alicloud Log
 2. Search for Alibaba Action Trail Event Collector.
 3. Click **Add instance** to create and configure a new integration instance.
 
```

Comment:
Hi @ShirleyDenkberg,
Item 1 is not correct as the pack is located somewhere else.
It should be:
1. Navigate to Settings > Configurations > Automation & Feed Integrations